### PR TITLE
Remove SDK plugin CMake variable

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

File: cmake/f3dConfig.cmake.in [hunk 1]
```diff
@@ -12,7 +12,7 @@
 #   f3d_MODULE_EXTERNAL_RENDERING  Will be enabled if F3D was built with external rendering support
 #   f3d_MODULE_RAYTRACING          Will be enabled if F3D was built with raytracing support
 #   f3d_BINDINGS_PYTHON            Will be enabled if F3D was built with python bindings
-#   f3d_SDK_PLUGIN                 Will be enabled if F3D plugin SDK is available
```
Comment (mwestphal) [inline cmake/f3dConfig.cmake.in:15]: is there a doc somewhere that needs to be udpated ?

File: cmake/f3dConfig.cmake.in [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
+#   f3d_BINDINGS_JAVA            Will be enabled if F3D was built with java bindings
```
Comment (mwestphal) [inline cmake/f3dConfig.cmake.in:15]: not aligned

File: cmake/f3dConfig.cmake.in (hunk 1 above)
Comment (Meakk) [reply cmake/f3dConfig.cmake.in:15]: I couldn't find any doc

Comment (mwestphal) [reply cmake/f3dConfig.cmake.in:15]: added to the plugin issue

Comment (mwestphal) [reply cmake/f3dConfig.cmake.in:15]: https://f3d.app/doc/libf3d/PLUGINS.html

Comment (mwestphal) [reply cmake/f3dConfig.cmake.in:15]: it just do not mention the cmake var. well the doc is correct now then.

**Pull request merged**